src/Aop/Framework/TraitIntroductionInfo.php

```diff
@@ -19,48 +19,48 @@ class TraitIntroductionInfo implements IntroductionInfo
 {
 
     /**
-     * List of interfaces to introduce
+     * Introduced interface
      *
-     * @var array
+     * @var string
      */
-    private $introducedInterfaces;
+    private $introducedInterface = '';
 
     /**
-     * List of traits to include
+     * Trait to use
      *
-     * @var array
+     * @var string
      */
-    private $introducedTraits;
+    private $introducedTrait = '';
```

Comment:
All seams fine, I just can not figure out why did you set defaults to empty string?